examples/decomposition/plot_pca_3d.py

```diff
@@ -73,8 +73,6 @@ def plot_figs(fig_num, elev, azim):
     pca_score = pca.explained_variance_ratio_
     V = pca.components_
 
-    x_pca_axis, y_pca_axis, z_pca_axis = V.T * pca_score / pca_score.min()
```

Comment:
should this be the preferred version? 3 looks a bit magical.

Reply:
@jnothman I'm afraid not. 3 is to make sure that the area of the surface is reasonable. If we use the removed statement, we'll get very strange result (I remain the labels on the x-axis to further demonstrate it):
current version:
![index](https://user-images.githubusercontent.com/12003569/30312947-3774041a-97ce-11e7-92e1-966888d4e7ce.png)
strange version using removed statement:
![index2](https://user-images.githubusercontent.com/12003569/30312968-4b3c5f7e-97ce-11e7-9fe5-b36cfa6850f9.png)
It seems that the cause of the two statements is https://github.com/scikit-learn/scikit-learn/commit/43b962a21006b09a98b9b4219a328268fb6c1203, where we no longer use an old method and somehow keep part of its code.